### DISCUSSION TITLE
Feature request: move Isolate to the users tab

### DISCUSSION THREAD

**Discussion by daltonrooney** [trendyminds/isolate#34] [Ideas]
Last one for now :-)

It seems like it would make sense to have the ability to control a user's permissions while editing that user. So in addition to "Account" and "Permissions" on the user record, perhaps there could be an Isolate tab there as well.

**Comment by aaronbushnell** [comment]
Hello! We've moved all open Issues to **Discussions**.

Unfortunately, due to client workload we often find ourselves unable to implement new features or resolve issues in a timely manner. Switching to Discussions allows better categorization to separate "Feature Requests" from bugs. It also allows the community to better suggest/provide solutions when we're unable to do so.

Hope Isolate continues to be useful for you!
